cea/visualisation/c_plotter.py

```
@@ -0,0 +1,411 @@
+"""
+PlotManager – Generates the Plotly graph
+
+"""
+
+from cea.visualisation.format.plot_colours import COLOURS_TO_RGB, COLUMNS_TO_COLOURS
+from cea.visualisation.b_data_processor import demand_x_to_plot_building
+from cea.import_export.result_summary import month_names, month_hours, season_mapping, season_names
```

Ruff (0.8.2)

`cea.import_export.result_summary.month_hours` imported but unused

Remove unused import

(F401)